src/main/java/agh/ics/oop/GrassField.java

```diff
@@ -0,0 +1,60 @@
+package agh.ics.oop;
+
+import java.util.ArrayList;
+import java.util.stream.Stream;
+
+public class GrassField extends AbstractWorldMap{
+
+    private final int grassnum;
```

Comment:
zbędny atrybut, to zwykła zmienna lokalna, można ją przekazać do `randomizeGrass()` jako parametr